opencadd/structure/superposition/sequences.py

```diff
@@ -77,6 +77,27 @@ def sequence_alignment(seq1: str, seq2: str, matrix: str, gap: int, local: bool
     return alignment[0]
 
 
+def find_gap_character(seq):
+    """
+    Finds the gap character of an alignment sequence
+
+    Args:
+        seq: str
+            aligned sequence
+
+    Returns:
+        gap character if found: str
+        None else
+    """
+    if '-' in seq:
+        return '-'
+    elif '=' in seq:
+        return '='
+    else:
+        return ''
+    #raise ValueError("No standard gap character found!")
+
+
 def fasta2select(
```

Comment:
Shouldn't the gap characters be removed explicitly now that BioPython doesn't recognize them?

From [BioPython Documentation](https://biopython.org/wiki/Alphabet):

Another case where the alphabet was used was in declaring the gap character, by default `-` in the various Biopython sequence and alignment parsers. If you are using a different character, **you will need to pass this to the `Seq` object `.replace()` method explicitly now**:
```python
# Old style
from Bio.Alphabet import generic_dna, Gapped
from Bio.Seq import Seq

my_dna = Seq("ACGT=TT", Gapped(generic_dna, "="))
print(my_dna.ungap())
```

```python
# New style
from Bio.Seq import Seq

my_dna = Seq("ACGT=TT")
print(my_dna.replace("=", ""))
```